nautobot_golden_config/nornir_plays/config_intended.py

```diff
@@ -51,27 +57,31 @@ def run_template(  # pylint: disable=too-many-arguments
     intended_obj.intended_last_attempt_date = task.host.defaults.data["now"]
     intended_obj.save()
 
-    intended_path_template_obj = check_jinja_template(obj, logger, global_settings.intended_path_template)
-    output_file_location = os.path.join(intended_root_folder, intended_path_template_obj)
+    # Render output relative filepath and jinja template filenames
+    intended_output_filepath = render_jinja_template(obj, logger, global_settings.intended_path_template)
+    jinja_intended_template_filename = render_jinja_template(obj, logger, global_settings.jinja_path_template)
 
-    jinja_template = check_jinja_template(obj, logger, global_settings.jinja_path_template)
-
-    status, device_data = graph_ql_query(job_result.request, obj, global_settings.sot_agg_query)
+    output_file_location = os.path.join(intended_root_folder, intended_output_filepath)
+    status, device_data = graph_ql_query(nautobot_job.request, obj, global_settings.sot_agg_query)
     if status != 200:
         logger.log_failure(obj, f"The GraphQL query return a status of {str(status)} with error of {str(device_data)}")
         raise NornirNautobotException()
     task.host.data.update(device_data)
 
+    jinja_settings = Jinja2.get_default()
+    jinja_env = jinja_settings.env
```

Comment:
Presumably, the point being one should use the jinja standard methods to add filters?

Reply:
django_jinja standard methods; docs will point this out

https://niwinz.github.io/django-jinja/latest/#_custom_filters_globals_constants_and_tests
https://niwinz.github.io/django-jinja/latest/#_registering_filters_in_a_django_way
https://niwinz.github.io/django-jinja/latest/#_complete_example